CHANGELOG.rst

```diff
@@ -4,6 +4,33 @@ Changelog
 
 This is the changelog for Puppetboard.
 
+0.2.1
```

Comment:
if SemVer is used, this IMHO needs to be 0.3.0 as .z is only for bugfixes and "daily charts" or "fact graph chart now configurable" sound like new features.